src/main/kotlin/ecommerce/service/AuthService.kt

```diff
@@ -0,0 +1,57 @@
+package ecommerce.service
+
+import ecommerce.dto.RegisteredMember
+import ecommerce.dto.Role
+import ecommerce.dto.TokenRequest
+import ecommerce.dto.TokenResponse
+import ecommerce.exception.ConflictException
+import ecommerce.exception.ForbiddenException
+import ecommerce.exception.NotFoundException
+import ecommerce.exception.UnauthorizedException
+import ecommerce.infrastructure.JwtTokenProvider
+import ecommerce.model.Member
+import ecommerce.repository.MemberRepository
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Transactional
+@Service
+class AuthService(
+    private val jwtTokenProvider: JwtTokenProvider,
+    private val memberRepository: MemberRepository,
+) {
+    fun findMember(payload: String): RegisteredMember {
+        val member = memberRepository.findByEmail(payload)
+        if (member == null) {
+            throw NotFoundException("Member not found")
+        }
```

Comment:
```suggestion
        val member = memberRepository.findByEmail(payload) ?: throw NotFoundException("Member not found")
```

What about using the Elvis operator to make it more idiomatic Kotlin?

Reply:
Oh, I applied this part in this [commit](https://github.com/herotechcourse/spring-ecommerce-order/pull/15/commits/6f0710336f138c1c2809c51c39da60ec41311e74).